akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -458,6 +498,19 @@ class ShardRegion(
     }
   }
 
+  def receiveRegionStateQuery(ref: ActorRef): Unit = {
+    import akka.pattern.{ ask, pipe }
+    import scala.concurrent.duration._
+    implicit val timeout: Timeout = 1.seconds
```

Comment:
I agree that this is not important enough to be defined in config, but I think we could increase it to at least 3.seconds ?